### ISSUE TITLE
Unrecognized option found "pylint.extensions.docstyle"

### ISSUE THREAD

**Issue by Mahrkeenerh** [PyCQA/pylint#7250]
### Bug description

After updating pylint, it's not able to load docstring style `Numpy`.

Nothing changed with my files, just the `pip install --upgrade pylint`

### Configuration

```ini
# DOCSTYLE
pylint.extensions.docstyle=Numpy
```

### Pylint output

```shell
.pylintrc:1:0: E0015: Unrecognized option found: pylint.extensions.docstyle (unrecognized-option)
```

### Pylint version

```shell
pylint 2.14.5
astroid 2.11.7
Python 3.10.2 (tags/v3.10.2:a58ebcc, Jan 17 2022, 14:12:15) [MSC v.1929 64 bit (AMD64)]
```

### OS / Environment

happened in github, and vs code terminal

**Comment by Pierre-Sassoulas** [comment]
2.14 got better at warning when the configuration file has issues. This failed silently previously, as this option does not exists. In fact we auto detect the docstring format and handle multiple docstring format if you use  in the same code base so you don't need any option if you activate the docparams checker https://pylint.pycqa.org/en/latest/user_guide/checkers/extensions.html#pylint-extensions-docparams with ``load-plugins = pylint.extensions.docparams``.

**Comment by Mahrkeenerh** [comment]
thank you
